dbdev/models.py

```diff
@@ -0,0 +1,45 @@
+import psycopg2
+
+from flask import abort,Response
+
+
+connect = psycopg2.connect("dbname='ridemyway' host='localhost' user='dario' password='riot'")
+curs = connect.cursor()
+
+
+
+#Class to add new users to the db
+class DbManager(object):
```

Comment:
Hey, have you checked the importance of having a docstring comments on your classes, functions and also the modules?